pkgs/development/python-modules/qcodes-contrib-drivers/default.nix

```diff
@@ -12,7 +12,7 @@
   pytestCheckHook,
   pytest-mock,
   pyvisa-sim,
-  stdenv,
+  stdenvNoCC,
```

Comment:
stdenv is already a dependency of buildPythonPackage

Reply:
> I think the intended meaning was “there’s no point to using `stdenvNoCC` rather than `stdenv` because the machinery this package uses already depends on the full `stdenv`”, not that you can omit the argument entirely.

Same intention from https://github.com/NixOS/nixpkgs/pull/340596  btw

Reply:
Got it. I was asked to use `stdenvNoCC` in another PR yesterday. Perhaps it's per reviewers' preference? Happy to set it either way.